### DISCUSSION TITLE
Welcome to proxybroker2 Discussions!

### DISCUSSION THREAD

**Discussion by bluet** [bluet/proxybroker2#62] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by afunTW** [comment]
Hi there,

Regarding the lack of developer and the maintenance requirement, I would like to get more involved in this project. Take the PR for example, we can see the first incoming issue is how to maintain the new Python version, especially from 3.8 to 3.10. There is some breaking change on the async io module.

One more discussion, should we keep Python 3.6 support?
